README.md

```diff
@@ -5,8 +5,8 @@ language that you can read [here](https://rust-unofficial.github.io/patterns/).
 
 ## Contributing
 
-You are missing content in this repository that can be helpful for others and
-you are eager to explain it? Awesome! We are always happy about new contributions
+Are you eager to eager to contribute missing content?
```

Comment:
I think it might still be better to have "helpful for others", seemed like a plus to keep that.

Reply:
> Are you eager to eager to contribute missing content _that is helpful for others_?